plugins/services/services_inspector.rb

```diff
@@ -51,24 +51,55 @@ def summary
   private
 
   def inspect_systemd_services
-    output = @system.run_command(
+    services = list_unit_files.map do |line|
+      name, state = line.split(/\s+/)
+
+      Service.new(name: name, state: state)
+    end
+
+    list_unit_files(true).map { |line| line.split(/\s+/) }.each do |name, state|
+      services << Service.new(name: name, state: state) if state == "disabled"
+
+      prefix, suffix = name.split("@")
+      list_unit_instances.select { |i| i =~ /#{prefix}.*#{suffix}/ }.each do |instance|
```

Comment:
Should we maybe include the `@` in the regex in order to make sure to map a wrong service?